recipe/meta.yaml

```diff
@@ -1,37 +1,31 @@
 {% set name = "eth-hash" %}
-{% set version = "0.5.1" %}
-
+{% set version = "0.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9805075f653e114a31a99678e93b257fb4082337696f4eff7b4371fe65158409
+  sha256: bacdc705bfd85dadd055ecd35fd1b4f846b671add101427e089a4ca2e8db310a
 
 build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<39]
 
 requirements:
   host:
+    - python
     - pip
-    - python >=3.5,<4.0
-    - setuptools-markdown
-    - pycryptodome
+    - wheel
+    - setuptools
   run:
-    - python >=3.5,<4.0
-    - pycryptodome >=3.6.6,<4
-  run_constrained:
-    - pycryptodome >=3.6.6,<4
-    - pysha3 >=1.0.0,<2.0.0
```

Comment:
@AndrewOsip As these packages are optional **AND** they have version constraints, you can move them to:
```
  run_constrained:
    - pycryptodome >=3.6.6,<4
    - pysha3 >=1.0.0,<2.0.0  # [py<39]
    - safe-pysha3 >=1.0.0     # [py>=39]
```
then if a user wants to install them in the environment, these packages will have these constraints only

Reply:
Is it ok with you if we follow what conda-forge did, moving the `pycryptodome >=3.6.6,<4` in the run as a default backend?